<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="SimuladorView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>1200</width>
      <height>490</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade de veículos que entraram no sistema:</string>
    </property>
   </widget>
   <widget class="QLabel" name="qtd_in">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>13</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade de veículos que saíram do sistema:</string>
    </property>
   </widget>
   <widget class="QLabel" name="qtd_out">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Tempo de simulação (em segundos): </string>
    </property>
   </widget>
   <widget class="QLabel" name="timer">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>17</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSobre">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>140</y>
      <width>148</width>
      <height>69</height>
     </rect>
    </property>
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionSobre"/>
   </widget>
   <addaction name="menuSobre"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionIniciar"/>
   <addaction name="actionPausar"/>
   <addaction name="actionParar"/>
   <addaction name="action10x"/>
   <addaction name="action100x"/>
   <addaction name="action_10x"/>
   <addaction name="action_100x"/>
  </widget>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionIniciar">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/Imagens/GUI/Buttons/media-playback-start.svg</normaloff>:/Imagens/GUI/Buttons/media-playback-start.svg</iconset>
   </property>
   <property name="text">
    <string>Iniciar</string>
   </property>
  </action>
  <action name="actionPausar">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/Imagens/GUI/Buttons/media-playback-pause.svg</normaloff>:/Imagens/GUI/Buttons/media-playback-pause.svg</iconset>
   </property>
   <property name="text">
    <string>Pausar</string>
   </property>
  </action>
  <action name="actionParar">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/Imagens/GUI/Buttons/media-playback-stop.svg</normaloff>:/Imagens/GUI/Buttons/media-playback-stop.svg</iconset>
   </property>
   <property name="text">
    <string>Parar</string>
   </property>
  </action>
  <action name="action10x">
   <property name="text">
    <string>+10x</string>
   </property>
  </action>
  <action name="action100x">
   <property name="text">
    <string>+100x</string>
   </property>
  </action>
  <action name="action_10x">
   <property name="text">
    <string>-10x</string>
   </property>
  </action>
  <action name="action_100x">
   <property name="text">
    <string>-100x</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SimuladorView</class>
   <extends>QGraphicsView</extends>
   <header location="global">simuladorview.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections/>
</ui>
